services/order.py

```diff
@@ -0,0 +1,28 @@
+from db.models import MovieSession, Order, Ticket
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: str | None = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date

```

Comment:
The manual assignment of the `created_at` field is not typical, as this field is usually managed automatically by Django. Ensure that this manual assignment is necessary and correctly formatted.